### ISSUE TITLE
[Magnum] kubectl logs cannot open file

### ISSUE THREAD

**Issue by zonca** [zonca/jupyterhub-deploy-kubernetes-jetstream#29]
`kubectl logs` doesn't work at all, for none of the pods.

Error is:

```
failed to open log file "/var/log/pods/1f0e29fa-7f6c-11ea-9bbc-fa163eef9c2b/hub/0.log": open /var/log/pods/1f0e29fa-7f6c-11ea-9bbc-fa163eef9c2b/hub/0.log: no such file or directory
```

The problem is that the file is there.

**Comment by zonca** [comment]
Those are symlinks to `/var/lib/docker/`, and was readable only by root, tried to make everything readable. But still broken.

**Comment by zonca** [comment]
## Workaround

* `logs.sh`: `ssh fedora@$IP sudo tail -30 $1`
* `logsw.sh` : `ssh fedora@$IPW sudo tail -30 $1`

**Comment by zonca** [comment]
fixed in #30
